contributing.md

```diff
@@ -0,0 +1,153 @@
+|`href`|string|Yes| The URL segment to use for the page. If there is a corresponding Markdown file, `href` must match the name of the Markdown file, minus the file extension.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+|`title`|string|Yes| The title of the page, used as the page name within the TOC on the left.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|`excerpt`|string|No| The short description of the page, used for the page card if `pageComponents` is set to `featured-cards`. See `pageComponents` for details.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Point out that this MUST be the same as the excerpt in the page metadata.

Reply:
It cannot be the same because the limits are different. Content cards can only accommodate around 100 characters, while the SEO description should be up to 200. I will specify the character limit though.